R/backend-sqlite.R

```diff
@@ -43,11 +43,12 @@ sql_query_explain.SQLiteConnection <- function(con, sql, ...) {
 }
 
 #' @export
-sql_query_set_op.SQLiteConnection <- function(con, x, y, method, ..., all = FALSE) {
+sql_query_set_op.SQLiteConnection <- function(con, x, y, method, ..., all = FALSE, lvl = 0) {
   # SQLite does not allow parentheses
+  method <- paste0(method, if (all) " ALL")
   build_sql(
     x,
-    "\n", sql(method), if (all) sql(" ALL"), "\n",
+    "\n", indent_lvl(sql_kw(method), lvl = lvl), "\n",
```

Comment:
`sql_kw()` is not really necessary but it might useful to wrap SQL keywords in a method, e.g. for coloured output.
Currently, this is not yet consistent in this PR so depending on what you think I either remove `sql_kw()` or wrap things consistently with `sql_kw()`